app/models/vm_or_template/operations.rb

```diff
@@ -118,6 +118,16 @@ def validate_vm_control
       msg ||= _("Provider doesn't support vm_destroy") unless ext_management_system.respond_to?(:vm_destroy)
       unsupported_reason_add(:terminate, msg) if msg
     end
+
+    supports :suspend do
+      msg = unsupported_reason(:control) unless supports_control?
+      msg ||= _("The VM is not powered on") unless vm_powered_on?
+      unsupported_reason_add(:suspend, msg) if msg
+    end
+  end
+
+  def vm_powered_on?
+    current_state == "on"
```

Comment:
looks it was my miss typo(late evening bad time for patches) so do not pay attention to the comment above

only problem that I have now connect to the line 
_("Provider doesn't support vm_suspend") unless ext_management_system.respond_to?(:vm_suspend)

Failed examples:

rspec ./spec/requests/api/instances_spec.rb:261 # Instances API Instance suspend action suspends an instance
rspec ./spec/requests/api/instances_spec.rb:269 # Instances API Instance suspend action suspends multiple instances
rspec ./spec/models/manageiq/providers/redhat/infra_manager/vm_spec.rb[1:1:3:1] # ManageIQ::Providers::Redhat::InfraManager::Vm#is_available? with :suspend when powered on